services/notify/src/main.js

```diff
@@ -52,6 +54,22 @@ const load = loader({
     }),
   },
 
+  BlacklistedNotification: {
+    requires: ['cfg', 'monitor', 'process'],
+    setup: ({cfg, monitor, process}) => data.BlacklistedNotification.setup({
+      tableName: cfg.app.blacklistedNotificationTableName,
+      credentials: sasCredentials({
+        accountId: cfg.azure.accountId,
+        tableName: cfg.app.blacklistedNotificationTableName,
+        rootUrl: cfg.taskcluster.rootUrl,
+        credentials: cfg.taskcluster.credentials,
+      }),
+      cryptoKey: cfg.azure.cryptoKey,
+      signingKey: cfg.azure.signingKey,
```

Comment:
These two can be omitted, since we are neither encrypting nor signing any columns in this table.